app/src/androidTest/java/com/github/se/cyrcle/ui/testEnd2End/MainActivityTest.kt

```diff
@@ -241,7 +241,6 @@ class MainActivityTest {
 
     fun assertMapScreen() {
       composeTestRule.onNodeWithTag("MapScreen").assertIsDisplayed()
-      composeTestRule.onNodeWithTag("addButton").assertIsDisplayed().assertHasClickAction()
```

Comment:
We should add as a task for next sprint to implement the sign in/guest aspects of the app in the tests so that instead of deleting this line we could create more relevant tests

Reply:
Here it's ok there is already another test that check the exact same assertion. Once with a user logged-out, once with a user logged-in

Reply:
<img width="437" alt="image" src="https://github.com/user-attachments/assets/3802e7eb-9371-4fa6-a2c0-49071cb81adf">

Reply:
I agree that functionally speaking the button is asserted correctly multiple times. However, we should try to make the E2E tests as complete and independant as possible (and here we do not assert that the button does not exist in the E2E tests). Again, this is just a minor detail that we can adress later. That's a non blocking issue for me.